From 732cc99c4d25b741e91b5e2f906a9fe70d75322e Mon Sep 17 00:00:00 2001
From: must_eat <mss@tutanota.de>
Date: Thu, 16 May 2024 23:27:37 +0300
Subject: [PATCH 5/8] clang: disable multiarch layout.

---
 clang/lib/Driver/ToolChains/Linux.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/clang/lib/Driver/ToolChains/Linux.cpp b/clang/lib/Driver/ToolChains/Linux.cpp
index 4300a2bdf..2159afb90 100644
--- a/clang/lib/Driver/ToolChains/Linux.cpp
+++ b/clang/lib/Driver/ToolChains/Linux.cpp
@@ -47,6 +47,8 @@ std::string Linux::getMultiarchTriple(const Driver &D,
   bool IsMipsR6 = TargetTriple.getSubArch() == llvm::Triple::MipsSubArch_r6;
   bool IsMipsN32Abi = TargetTriple.getEnvironment() == llvm::Triple::GNUABIN32;
 
+  return "";
+
   // For most architectures, just use whatever we have rather than trying to be
   // clever.
   switch (TargetTriple.getArch()) {
-- 
2.45.0

